### PR TITLE
Dao contracts documentation

### PR DESCRIPTION
**Pull request by MiltonTulli**
### Description
Improve [`DAOController`](https://github.com/MiltonTulli/dxdao-contracts/blob/feaure/1.5-documentation/docs/contracts/dao/DAOController.md), [`DAOAvatar`](https://github.com/MiltonTulli/dxdao-contracts/blob/feaure/1.5-documentation/docs/contracts/dao/DAOAvatar.md),  [`DAOReputation`](https://github.com/MiltonTulli/dxdao-contracts/blob/feaure/1.5-documentation/docs/contracts/dao/DAOReputation.md), [`DXDVotingMachine`](https://github.com/MiltonTulli/dxdao-contracts/blob/feaure/1.5-documentation/docs/contracts/dao/votingMachine/DXDVotingMachine.md) & [`Scheme`](https://github.com/MiltonTulli/dxdao-contracts/blob/feaure/1.5-documentation/docs/contracts/dao/schemes/Scheme.md) contracts documentation.

Refactor comments, arguments, and returns definitions.

https://github.com/MiltonTulli/dxdao-contracts/blob/feaure/1.5-documentation/docs/SUMMARY.md

### CODE REVIEW

File: contracts/dao/DAOController.sol [hunk 1]
```diff
@@ -175,31 +201,35 @@ contract DAOController is Initializable {
             }
             schemesWithManageSchemesPermission = schemesWithManageSchemesPermission - 1;
         }
+        delete schemes[_scheme];
```
Comment (MiltonTulli) [inline contracts/dao/DAOController.sol:204]: Moving this to emit event after scheme was deleted. Also to be consistent with registerScheme fn

File: contracts/dao/votingMachine/DXDVotingMachine.sol [hunk 2]
```diff
@@ -379,17 +384,14 @@ contract DXDVotingMachine {
...
-     *           [1] proposerReputationReward
+     * @dev Redeem a reward for a successful stake, vote or proposing.
+     *      The function use a beneficiary address as a parameter (and not msg.sender) to enable users to redeem on behalf of someone else.
+     * @param _proposalId The ID of the proposal
+     * @param _beneficiary The beneficiary address
+     * @return rewards [0]=stakerTokenReward [1]=proposerReputationReward
      */
     // solhint-disable-next-line function-max-lines,code-complexity
-    function redeem(bytes32 _proposalId, address _beneficiary) public returns (uint256[3] memory rewards) {
+    function redeem(bytes32 _proposalId, address _beneficiary) public returns (uint256[2] memory rewards) {
```
[the remark this answers is not in the document]
Comment (AugustoL) [reply contracts/dao/votingMachine/DXDVotingMachine.sol:394]: Nope, looks good.

File: contracts/dao/votingMachine/DXDVotingMachine.sol [hunk 3]
```diff
@@ -83,28 +82,29 @@ contract DXDVotingMachine {
...
 
     struct Proposal {
-        bytes32 schemeId; // the scheme unique identifier the proposal is target to.
-        address callbacks; // should fulfill voting callbacks interface.
+        bytes32 schemeId; // The scheme unique identifier the proposal is target to.
+        address callbacks; // Should fulfill voting callbacks interface.
         ProposalState state;
         ExecutionState executionState;
-        uint256 winningVote; //the winning vote.
+        uint256 winningVote; // She winning vote.
```
Comment (AugustoL) [inline contracts/dao/votingMachine/DXDVotingMachine.sol:94]: The

File: contracts/dao/votingMachine/DXDVotingMachine.sol [hunk 4]
```diff
@@ -1121,22 +1117,22 @@ contract DXDVotingMachine {
     }
 
     /**
-     * @dev _score return the proposal score (Confidence level)
+     * @dev Return the proposal score (Confidence level)
```
Comment (AugustoL) [inline contracts/dao/votingMachine/DXDVotingMachine.sol:1120]: Returns

File: contracts/dao/votingMachine/DXDVotingMachine.sol [hunk 5]
```diff
@@ -1302,68 +1299,68 @@ contract DXDVotingMachine {
...
      */
     function getProposalTimes(bytes32 _proposalId) external view returns (uint256[3] memory times) {
         return proposals[_proposalId].times;
     }
 
     /**
-     * @dev getProposalScheme return the schemeId for a given proposal
-     * @param _proposalId the ID of the proposal
-     * @return bytes32 scheme identifier
+     * @dev Return the schemeId for a given proposal
```
Comment (AugustoL) [inline contracts/dao/votingMachine/DXDVotingMachine.sol:1311]: ReturnS

**Review by AugustoL — CHANGES_REQUESTED** [review]

File: contracts/dao/votingMachine/DXDVotingMachine.sol (hunk 3 above)
Comment (MiltonTulli) [reply contracts/dao/votingMachine/DXDVotingMachine.sol]: Fixed!

File: contracts/dao/votingMachine/DXDVotingMachine.sol (hunk 4 above)
Comment (MiltonTulli) [reply contracts/dao/votingMachine/DXDVotingMachine.sol]: Fixed!

File: contracts/dao/votingMachine/DXDVotingMachine.sol (hunk 5 above)
Comment (MiltonTulli) [reply contracts/dao/votingMachine/DXDVotingMachine.sol]: Fixed all "return" typos. Than you

**Review by AugustoL — APPROVED** [review]

**Pull request merged**